tonic/models/vic/grid_params.py

```diff
@@ -426,18 +576,48 @@ def _run(args):
     nc_file = make_grid(grid_file=args.grid_file,
                         soil_file=args.soil_file,
                         snow_file=args.snow_file,
-                        veg_file=args.veg_file,
                         vegl_file=args.vegl_file,
+                        veg_file=args.veg_file,
+                        lake_file=args.lake_file,
                         nc_file=args.out_file,
-                        version=args.VIC_version)
-
-    print('completed grid_parms.main(), output file was: {0}'.format(nc_file))
+                        version_in=args.VIC_version_in,
+                        grid_decimal=args.grid_decimal,
+                        nlayers=args.nlayers,
+                        snow_bands=args.snow_bands,
+                        veg_classes=args.veg_classes,
+                        max_roots=args.max_roots,
+                        max_numnod=args.max_numnod,
+                        cells=args.cells,
+                        organic_fract=args.organic_fract,
+                        spatial_frost=args.spatial_frost,
+                        spatial_snow=args.spatial_snow,
+                        july_tavg_supplied=args.july_tavg_supplied,
+                        veglib_fcan=args.veglib_fcan,
+                        blowing_snow=args.blowing_snow,
+                        vegparam_lai=args.vegparam_lai,
+                        vegparam_fcan=args.vegparam_fcan,
+                        vegparam_albedo=args.vegparam_albedo,
+                        lai_src=args.lai_src,
+                        fcan_src=args.fcan_src,
+                        alb_src=args.alb_src,
+                        lake_profile=args.lake_profile)
+
+    print('completed grid_params.main(), output file was: {0}'.format(nc_file))
 # -------------------------------------------------------------------- #
 
 
 # -------------------------------------------------------------------- #
-def make_grid(grid_file, soil_file, snow_file, veg_file, vegl_file,
-              nc_file='params.nc', version='4.1.2'):
+def make_grid(grid_file, soil_file, snow_file, vegl_file, veg_file,
+              lake_file, nc_file='params.nc', version_in='4.2',
+              grid_decimal=4, nlayers=3, snow_bands=5, veg_classes=11,
+              max_roots=3, max_numnod=10, cells=False,
+              organic_fract=False, spatial_frost=False,
+              spatial_snow=False, july_tavg_supplied=False,
+              veglib_fcan=False, blowing_snow=False,
+              vegparam_lai=False, vegparam_fcan=False,
+              vegparam_albedo=False, lai_src=FROM_VEGLIB,
+              fcan_src=FROM_DEFAULT, alb_src=FROM_VEGLIB,
+              lake_profile=False):
```

Comment:
As stated above, I'd prefer to use strings instead of the integer representation of the vegetation options (e.g. `FROM_DEFAULT` becomes `'FROM_DEFAULT'`.